src/views/Simulation/Resultats.vue

```diff
@@ -164,6 +165,8 @@ export default {
           calculs.resultats.droitsEligibles.forEach(function (d) {
             vm.$matomo && vm.$matomo.trackEvent("General", "show", d.label)
           })
+          const id = this.$matomo ? this.$matomo.getVisitorId() : undefined
```

Comment:
Est ce que `$matomo` est disponible directement dans le `mixin` (via `this.$matomo` comme ici) plutôt que devoir le passer en paramètre ?

Reply:
Oui, c'est corrigé.